frontend/src/types/BookTypes.ts

```diff
@@ -30,14 +30,14 @@ export type Publisher = {
 export type Book = {
   title: string;
   coverImage: string;
-  titlePrefix: string;
-  seriesOrder: string;
-  illustrator: string[];
-  translator: string[];
+  titlePrefix: string | null;
+  seriesOrder: string | null;
+  illustrator: string[] | null;
```

Comment:
maybe this should be mapped to an empty string array instead of null? It would prevent UI components from having to check nullity

Reply:
done